backend/backend/ReactionTimeGame/Controllers/ReactionTimeController.cs

```diff
@@ -8,23 +8,9 @@ namespace backend.ReactionTimeGame.Controllers;
 [Route("[controller]")]
 public class ReactionTimeController : ControllerBase
 {
-
-    private readonly ILogger<ReactionTimeController> _logger;
-
-    public ReactionTimeController(ILogger<ReactionTimeController> logger)
-    {
-        _logger = logger;
-    }
-
-    [HttpGet(Name = "GetNumber")]
-    public ReactionTime Get()
+    [HttpGet]
+    public ActionResult<ReactionTime> Get()
     {
-
-        const int millisecondsInSecond = 1000;
-
-        const int minWait = 2 * millisecondsInSecond;
-        const int maxWait = 5 * millisecondsInSecond;
-
-        return new ReactionTime { MillisecondsToWait = Random.Shared.Next(minWait, maxWait) };
+        return new ReactionTime();
```

Comment:
Nežunau ar worth yra perkelt tokia maža funkcija, nu bet nepagadinta :D

Reply:
Jo, fair. Cia labiau del to, kad controllery nebutu logikos + galimai ateity kazka addint reiks, tai tada vistiek sita reiks padaryt

Reply:
As manau nieko tokio kad iskelta, be man pavadinimas klases yra truputuka neaiskus, koda paziurejus aisku ka daro, bet pavadinimas not sure ar tikrai tinka.

Also, manau cia jeigu jau iskelti tai butu geriau iskelti i service, o ne tiesiog sukurti klase kuri enkapsuliuoja int'a viena

Reply:
Jo, dabar ziurint, name'as galetu but aiskesnis. No clue kas yra service, i'll look in to it.